lib/debug/ProfilingPlugin.js

```diff
@@ -119,7 +120,8 @@ function createTrace(outputPath) {
 	return {
 		trace,
 		counter,
-		profiler
+		profiler,
+		fsStream
```

Comment:
For separation concerns it may be better to only expose an `end(callback)` method instead of the whole fsStream object.